executor/pom.xml

```diff
@@ -22,6 +23,18 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-resteasy</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jackson</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
What's the need for OpenAPI and swagger-ui on the Executor and Shard?

Reply:
@robpblake I think the executor and the shard will have rest api, wdyt?

Reply:
I can only see the need for a REST APIs on the Manager (same as we did for DaaS). I imagine there will be two on the Manager:

- Customer Facing API that people use to create things
- API that the Shard(s) use to fetch details of what they need to deploy
  - For the first iteration, we may not even expose this one as a REST API as at Manager and Shard are deployed as a single unit, so it may just be directly injected as a bean

I don't see there being a REST API on the Shard or Executor. What's the use case you're seeing for a REST API on Shard + Executor?